README.md

```diff
@@ -41,7 +41,9 @@ To run this application locally, with hot-reloading, run
 npm run dev
 ```
 
-The application will start on [http://localhost:7080](http://localhost:7080). Article pages are served at routes of the format  `/news/articles/:id` where id is the asset ID generated by the Content Management System. 
+The application will start on [http://localhost:7080](http://localhost:7080). 
+
+Article pages are served at routes of the format  `/news/articles/:id` where id is the asset ID generated by the Content Management System. 
```

Comment:
Thanks for updating the readme! It might be worth adding documentation that persian articles can also be accessed locally